cerbero/packages/package.py

```diff
@@ -434,6 +432,13 @@ class MetaPackage(PackageBase):
     @type resources_distribution: string
     @cvar user_resources: folders included in the .dmg for iOS packages
     @type user_resources: list
+    @cvar osx_framework_library: name and link as a (name, relative_path)
```

Comment:
A MetaPackage should not produce a macOS Framework, only the SDKPackage should.
What seems to be wrong and might have confused you is this:
  if isinstance(self.package, MetaPackage):
            packager = self._create_framework_bundle_packager()
            self._create_framework_bundle_layout(packager)
            self._create_framework_bundle_package(packager)

Reply:
This flag was set in Package AND SDKPackage before.
There are lot of incongruities in all this code.
According to where it is used, this is the place it should be.
If there are places where it is used and it should not be, well.... when we've got time it would be a good idea to refactor all this stuff, but for now we cannot fix the whole cerbero in one PR ;)